doc/stake-pool-operations/getConfigFiles_AND_Connect.md

```diff
@@ -44,6 +44,9 @@ Get the complete list of available options with `cardano-node run --help`
   	                                specified slot
     -h,--help                       Show this help text
 
+**Note**: If you specify a host address for IPv4, but do not specify one for IPv6,
+your node will **NOT** be able to connect over IPv6, and vice versa.
+
```

Comment:
```suggestion
**Note**: If you don't specify `--host-addr` nor `--host-ipv6-addr`, node will use the default ipv4 and ipv6 addresses (depending which addresses are available).  If one specifies one of them only that address will be used, in particular if you only provide an IPv4 address, the node will not connect over IPv6.
```

I think it's better to be a little bit more descriptive and phrase it to suggest why the api works the way it does. It's not about not being able to connect over ipv6 but rather that you can precisely express which addresses the node can use.

Reply:
I've committed your suggestion. It still leaves open the question of specifying an IPv4 host address but relying on IPv6 privacy extension (RFC4941), where ideally you would not want to specify an IPv6 address, since they can change all the time.